query/metaquery/completers.go

```diff
@@ -31,7 +31,7 @@ func Complete(input *CompleterInput) []prompt.Suggest {
 
 func completerFromArgsOf(cmd string) completer {
 	return func(input *CompleterInput) []prompt.Suggest {
-		metaQueryDefinition, _ := metaQueryDefinitions[cmd]
+		metaQueryDefinition := metaQueryDefinitions[cmd]
```

Comment:
Is this a change that continues to ignore an error? If so, shouldn't we handle the error?

Reply:
`metaQueryDefinitions` is a map[string]metaQueryDefinition. We are just assigning the value here.

Reply:
to expand: in this map syntax ```metaQueryDefinition, _ := metaQueryDefinitions[cmd]``` the second return val is a boolean indicating whether the value was found